js/public/bindings.js

```diff
@@ -41,25 +44,26 @@ const setBinding = (e, values) => {
         delete bindings[oldId[1]][oldId[2]]
     }
     binding.id = `bind-${type}-${idx}`
-
     bindings[type][idx] = {
         name: getVal('name'),
         invert: getVal('invert') === 'true',
-        action: getVal('action'),
+        action: getVal('action')
     }
 }
-
 const addBinding = values => {
     const labelInput = ({ label, elem, onChange, onInput, ...props }) => {
         const el = document.createElement('label')
-        el.innerText = label
+        if (label) el.innerText = label
```

Comment:
In case label is undefined, it would cast to string and actually be visible as the element's innerText. That's useful for debugging, so not good to check against.
One of the reasons I consider typescript can be damaging to code quality.